### DISCUSSION TITLE
Running OSX plugin server + plugin on same host strange behaviour?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [apohl79/audiogridder#858] [Help]
<img width="1512" alt="Screenshot 2021-12-18 at 18 02 28 1" src="https://user-images.githubusercontent.com/90145421/146649526-f9aab0e3-1175-4a5e-a6c3-bc979f38188f.png">

Finally managed to get it working after reinstalling everything.
Now I have the OSX server + OSX plugin beta 6.

Is the behaviour of the sever screen 'mirroring' normal? Its not possible to load the server part out of view?
IS this normal behaviour or should I move it to another computer.

**Comment by emack999** [comment]
Disable the screen capturing on the server settings.

![865](https://user-images.githubusercontent.com/81657685/146652618-97b52f3e-4d87-47c1-a89c-868c98350e2b.jpg)

**Reply by Jarrodsz** [reply]
Great awesome

Really must thank you for creating audiogridder. Do you accept donations? 🙏

**Comment by CrossGridder** [comment]
It seems someone found CrossGridder useful, isn't it? I am happy to see people using it... However, did you change anything? because the settings were changed from the default ones in CrossGridder if you experienced this behavior.
